modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemander.java

```diff
@@ -1142,6 +1185,13 @@ private String demandRoutineInfo(UUID supplier, GridDhtPartitionSupplyMessage su
             ctx = grp.shared();
 
             cancelLock = new ReentrantReadWriteLock();
+
+            if (!availablePrintRebalanceStatistics())
+                stat = null;
+            else {
+                CacheGroupRebalanceStatistics prevStat = previous.stat;
+                stat = new CacheGroupRebalanceStatistics(nonNull(prevStat) ? prevStat.attempt() + 1 : 1);
```

Comment:
I offer do that through ChaceSharedContext.

Reply:
What exactly should I do?

Reply:
I think we don't need to store it in context yet, so we can discuss it with core reviewer.